pkgs/tools/networking/sipexer/default.nix

```
@@ -0,0 +1,26 @@
+{ buildGoModule
+, fetchFromGitHub
+, lib
+}:
```

Can you put `lib` first in the `callPackage` args? Looks good otherwise, thank you!

```suggestion
{ lib
, buildGoModule
, fetchFromGitHub
}:
```